tests/python/unittest/test_top_level3.py

```diff
@@ -16,8 +16,15 @@ def test_leaky_relu():
     y = sym.leaky_relu(x, alpha=0.1)
     assert(y.list_input_names() == ["x"])
 
+def test_prelu():
```

Comment:
add a test case that automatically infer the shape of alpha for given  data shape

Reply:
infershape is also tested as part of the test_prelu_nchw and test_prelu_nhwc in compiler/test_top_level1.py. Any shape mismatch, will throw error as part of this testcase. Please check whether this is sufficient or i need to add another testcase.

Reply:
@tqchen still need to add testcase for this? please check whether  test_prelu_nchw and test_prelu_nhwc in compiler/test_top_level1.py is fine? since it already foes the shape mismatch validation for data n alpha